### PR TITLE
Store with Multi Files

### PR DESCRIPTION
**Pull request by moskoweb**
Using the normal single file function ends up bursting or causing slow recording of large amounts of conversations and messages, splitting to multiple files has worked better for storing information.

### CODE REVIEW

File: src/Store/make-in-memory-store.ts [hunk 1]
```diff
@@ -387,6 +387,53 @@ export default (
 				const json = JSON.parse(jsonStr)
 				fromJSON(json)
 			}
+		},
+		writeToMultiFiles: (folder: string) => {
+			// require fs here so that in case "fs" is not available -- the app does not crash
+			const { existsSync, mkdirSync, writeFileSync } = require('fs')
+			if (folder.includes('.json')){
```
Comment (adiwajshing) [inline src/Store/make-in-memory-store.ts:394]: isn't this redundant?

File: src/Store/make-in-memory-store.ts [hunk 2]
```diff
@@ -387,6 +387,53 @@ export default (
...
+		},
+		readFromMultiFiles: (folder: string) => {
+			// require fs here so that in case "fs" is not available -- the app does not crash
+			const { readFileSync, existsSync } = require('fs');
+			if (folder.includes('.json')){
+				readFromFile(folder);
+			} else {
+				if (existsSync(folder)) {
+					logger.debug({ folder }, 'reading from file');
+					const chats = JSON.parse(
```
Comment (adiwajshing) [inline src/Store/make-in-memory-store.ts:420]: if one of these reads fails -- won't it cause a crash?

File: src/Store/make-in-memory-store.ts (hunk 1 above)
Comment (moskoweb) [reply src/Store/make-in-memory-store.ts:394]: I don't see how it is, this validation is to make the migration from Single File to Multiple Files easier.

File: src/Store/make-in-memory-store.ts (hunk 2 above)
Comment (moskoweb) [reply src/Store/make-in-memory-store.ts]: I put verification of the files

**Comment by moskoweb** [comment]
Why has it become obsolete?

**Comment by amiruldev20** [comment]
this function can't be called when i try

**Comment by moskoweb** [comment]
> this function can't be called when i try

try again

**Comment by amiruldev20** [comment]
no work. error installed baileys

**Comment by moskoweb** [comment]
> no work. error installed baileys

What error appears to you?

**Pull request closed without merging**